docs/troubleshoot/ze/troubleshoot-ze.md

```diff
@@ -40,4 +40,38 @@ To fix the configuration file causing the error, follow these steps:
 2. Modify the file as needed and save the changes.
 3. Reload Visual Studio Code to confirm that Zowe Explorer can read the updated file.
 
+## Missing write access to VS Code `extensions` folder
 
+In some environments, the path for VS Code extensions (typically `~/.vscode/extensions`) can be read-only. This can happen when an environment has developers share a common read-only extensions folder that is managed by an admin with write access.
+
+In these cases, Zowe Explorer fails to activate because it cannot write to the `logs` and `temp` folders in the extension path. When Zowe Explorer launches, an error message displays, as in the following examples.
+
+`logs` folder write access error:
+
+![Logs folder write access error](../../images/troubleshoot/ZE/write-access-error-logs-folder.png)
+
+`temp` folder write access error:
+
+![Logs folder write access error](../../images/troubleshoot/ZE/write-access-error-temp-folder.png)
+
+To avoid this, change the `logs` and `temp` folder locations.
+
+Follow these steps:
+
+1. In Zowe Explorer, select the **File menu**, select **Preferences**, and click on **Settings**.
+
+2. In either the **User** or **Workspace** tab, click on the **Extensions** option to open the menu.
```

Comment:
Any reason why we cannot list only one option? What is the most common method?

Reply:
VS Code allows settings to be defined across different scopes ([source](https://code.visualstudio.com/docs/getstarted/settings)):
> * User Settings - Settings that apply globally to any instance of VS Code you open.
> * Workspace Settings - Settings stored inside your workspace and only apply when the workspace is opened.

The settings for logs and temp folders could be changed in either place - either globally for all VS Code instances, or only for the current project/workspace. However I think we could omit this part: "in either the **User** or **Workspace** tab".

Either the user should know enough to select a scope without us telling them, or if they don't then setting it globally is probably what they want, and is the default behavior because the most common tab that's selected by default is the first one (User).

Reply:
Leaving as it b/c the intention is to include the tabs as a wayfinder for the Extensions tab. It's not so much for the user to figure out which option they want, but it's meant as an indication of the area where they can find the Extensions tab.